README.md

```diff
@@ -1,12 +1,20 @@
+[![Pub Version](https://img.shields.io/pub/v/rohd)](https://pub.dev/packages/rohd/versions)
+[![Popularity](https://img.shields.io/pub/popularity/rohd)](https://pub.dev/packages/rohd/score)
 [![Tests](https://github.com/intel/rohd/actions/workflows/general.yml/badge.svg?event=push)](https://github.com/intel/rohd/actions/workflows/general.yml)
-[![API Docs](https://img.shields.io/badge/API%20Docs-generated-success)](https://intel.github.io/rohd/rohd/rohd-library.html)
 [![Chat](https://img.shields.io/discord/1001179329411166267?label=Chat)](https://discord.gg/jubxF84yGw)
 [![License](https://img.shields.io/badge/License-BSD--3-blue)](https://github.com/intel/rohd/blob/main/LICENSE)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/intel/rohd/blob/main/CODE_OF_CONDUCT.md)
 
+![ROHD Logo](doc/website/assets/images/rohd_logo.svg)
+
 Rapid Open Hardware Development (ROHD) Framework
 ================================================
 
+[![API Docs](https://img.shields.io/badge/API%20Docs-generated-success)](https://intel.github.io/rohd/api)
```

Comment:
Why are these two below the title and the rest are up on top?

Reply:
As part of the work on [improving GitHub Codespaces configuration](https://github.com/intel/rohd/pull/256) it would be nice to add the `Open in GitHub Codespaces` badge to the new `README.md`:

```text
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=409325108)
```

It should work like this:

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=409325108)

Reply:
Thanks for the suggestion @chkyon, I agree all of them under the banner sounds good, and adding the codespaces badge in this PR sounds like the right place

Reply:
Yup, @chykon I think that a good idea! Let me change it.

Reply:
@mkorbel1 @chykon I made some changes. What do you think?

Reply:
Great! Overall, the new README.md seems to be much better than the overly bloated old one.